File: src/providers/chatgpt/orchestration.ts
/**
 * ChatGPT orchestration strategy.
 */
import { OrchestrationStrategy } from "../../core/etl.js";

export class ChatGPTOrchestrationStrategy extends OrchestrationStrategy {
  readonly MANIFEST_MAP: Record<string, string> = {
    "conversations.json": "chatgpt_conversations",
  };
}

File: context_use/etl/providers/chatgpt/orchestration.py
from __future__ import annotations

from context_use.etl.core.etl import OrchestrationStrategy


class ChatGPTOrchestrationStrategy(OrchestrationStrategy):
    """Discovers ChatGPT ETL tasks from extracted archive files.

    Looks for ``conversations.json`` at the root of the archive.
    """

    MANIFEST_MAP = {
        "conversations.json": "chatgpt_conversations",
    }
